tests/Agent/IntegrationTests/IntegrationTests/CSP/HighSecurityAndCustomAttributes.cs

```diff
@@ -58,7 +58,7 @@ public void Test()
             Assertions.TransactionTraceDoesNotHaveAttributes(unexpectedTransactionTraceAttributes, TransactionTraceAttributeType.User, transactionSample);
 
             var errorTrace = _fixture.AgentLog.GetErrorTraces().FirstOrDefault();
-            var errorEventPayload = _fixture.AgentLog.GetErrorEvents().FirstOrDefault();
+            var errorEventPayload = _fixture.AgentLog.GetErrorEventPayloads().FirstOrDefault();
```

Comment:
Could benefit from just using `GetErrorEvents`.   My goal is removing the usage of `GetErrorEvenPayloads` to the end of removing the method entirely.

Reply:
Agreed.